<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/PC-QUEREN/Pictures/user (1).png</normaloff>C:/Users/PC-QUEREN/Pictures/user (1).png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>721</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica CE 35 Thin</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Hotel</string>
     </attribute>
     <widget class="QTableView" name="hotelView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>711</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>90</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>120</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>10</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
     </widget>
     <widget class="QPushButton" name="ajoutAdminBtn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(164, 193, 255);
border: 1px solid blue; 
border-radius: 15px;
padding:5px;
</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rechercheHotelEdit">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: transparent;
    color: black;
	border: 1px solid black;
	border-radius:10px ;
}
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>30</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ressource.qrc">:/icon/C:/Users/PC-QUEREN/Downloads/chercher.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>370</y>
        <width>231</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid; 
border-radius: 10px;
padding: 6px;
</string>
      </property>
      <property name="text">
       <string>Exporter en pdf</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>410</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ressource.qrc">:/icon/C:/Users/PC-QUEREN/Downloads/telecharger.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>30</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ressource.qrc">:/icon/C:/Users/PC-QUEREN/Downloads/chercher.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>380</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ressource.qrc">:/icon/C:/Users/PC-QUEREN/Downloads/telecharger.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Type">
     <attribute name="title">
      <string>Gerant</string>
     </attribute>
     <widget class="QLineEdit" name="rechercheHotelEdit_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: transparent;
    color: black;
	border: 1px solid black;
	border-radius:10px ;
}
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>30</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ressource.qrc">:/icon/C:/Users/PC-QUEREN/Downloads/chercher.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="typeChambreView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>711</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>90</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>120</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>10</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
     </widget>
     <widget class="QPushButton" name="ajoutGerantBtn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(164, 193, 255);
border: 1px solid blue; 
border-radius: 15px;
padding:5px;
</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="disconnectAdminBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica CE 35 Thin</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid; 
border-radius: 10px;
padding: 6px;
</string>
    </property>
    <property name="text">
     <string>Deconnection</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>530</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressource.qrc">:/icon/C:/Users/PC-QUEREN/Downloads/connexion (3).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica CE 35 Thin</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(108, 135, 255);</string>
    </property>
    <property name="text">
     <string>Gestionnaire d'hotels</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
